frontend/src/components/footer.tsx

```diff
@@ -15,28 +17,78 @@ import RiceLogoURL from '../images/rice-logo.png'
 import OpenStaxURL from '../images/openstax-logo.png'
 
 const Funders = () => {
+    const isMobile = useMediaQuery('(max-width: 768px)');
```

Comment:
These methods are already available - i believe we have useIsMobileDevice() and srinivas added a useIsTabletDevice() in another PR which will get merged in shortly